pkg/manager/setconfig.go

```diff
@@ -19,7 +19,9 @@ import (
 	"github.com/onosproject/onos-config/pkg/events"
 	"github.com/onosproject/onos-config/pkg/store"
 	"github.com/onosproject/onos-config/pkg/store/change"
-	types "github.com/onosproject/onos-config/pkg/types/change/device"
+	devicechangetypes "github.com/onosproject/onos-config/pkg/types/change/device"
```

Comment:
call this `devicetypes` to match `networktypes` below

Reply:
same